src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -549,26 +585,23 @@ def test_lookup_job_states__batch_id__not_batch(self):
 
     @mock.patch(CLIENTS, get_mock_client)
     def test_lookup_job_states__job_id_list__ee2_error(self):
-        def mock_check_jobs(self, params):
-            raise Exception("Test exception")
+        exc = Exception("Test exception")
+        exc_message = str(exc)
+
+        def mock_check_jobs(params):
+            raise exc
 
         job_id_list = ALL_JOBS
         req_dict = make_comm_msg(STATUS, job_id_list, False)
 
-        TIME_NOW = 987654321
-        with mock.patch("time.time") as fake_time:
-            fake_time.return_value = TIME_NOW
-            with mock.patch.object(
-                MockClients, "check_jobs", side_effect=mock_check_jobs
-            ):
-                self.jc._handle_comm_message(req_dict)
```

Comment:
no need for this lovely mocking now that the ee2 error is reported  in an `error` field